### DISCUSSION TITLE
如何更新国内CDN的Waline包 || How to update the Waline package of domestic CDN

### DISCUSSION THREAD

**Discussion by kok777** [walinejs/waline#2424] [Ideas]
起因：我之前一直用的https://cdn.onmicrosoft.cn/ 这个UNPKG回源 但是近期已经转移到CF去了 速度慢下来了。
所以看国内还是库的，但就是包很慢 比如下边三个。
BootCDN: [www.bootcdn.cn](https://www.bootcdn.cn)
七牛云: [www.staticfile.org](https://www.staticfile.org)
字节跳动: [cdn.bytedance.com](https:///cdn.bytedance.com)
速度很快 但是waline包很老
我看了他们说明 这几个都是同步于https://github.com/cdnjs/cdnjs
技术有限以前从没搞过这个，不知道咋让 这几个包能更新到最新版。

---
Cause: I have been using https://cdn.onmicrosoft.cn/ for UNPKG back to the source, but recently it has been transferred to CF and the speed has slowed down.
So it seems that the domestic database is still available, but the package is very slow, such as the three below.
BootCDN: [www.bootcdn.cn](https://www.bootcdn.cn)
Qiniuyun: [www.staticfile.org](https://www.staticfile.org)
ByteDance: [cdn.bytedance.com](https:///cdn.bytedance.com)
It’s fast but the waline bag is very old.
I read their instructions. These are all synchronized with https://github.com/cdnjs/cdnjs
The technology is limited and I have never done this before. I don’t know how to update these packages to the latest version.

**Comment by zonefile** [comment]
Replace `unpkg.com` with `s4.zstatic.net/npm`

**Comment by Francklin** [comment]
可以试试webcache.cn，支持cdnjs, npm和github
